src/lib.rs

```diff
@@ -2,4 +2,89 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 
-include!(concat!(env!("OUT_DIR"), "/cuda_driver_bindings.rs"));
+use std::ffi::CString;
+use std::sync::{Mutex, Once};
+use std::borrow::BorrowMut;
+
+use futures::{
+    executor::block_on,
+};
+
+use tarpc::{
+    client,
+    context, 
+    tokio_serde::formats::Json,
+};
+
+mod service;
+
+pub type CUresult = ::std::os::raw::c_uint;
```

Comment:
this block should be pulled from the autogenerated code. Can you add a `// TODO(asalkeld): use auto-generated library bindings for these types rather than manually generated github issue #XYZ`? and then link XYZ to the github issue?